### PR TITLE
Bug 1284504, verify custom configuration is added to new paths

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1284504

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@bfallonf @tpoitras can you please review? Thanks!

**Comment by ahardin-rh** [comment]
@sdodson, do these changes look okay to you?

**Comment by adellape** [comment]
It should be noted that the link that Eduardo cites in the BZ (https://docs.openshift.com/enterprise/3.1/install_config/upgrades.html) has since been broken up into the "Upgrading" subdirectory with multiple topics. Whereas previously, it was a single long topic.

So this information is maybe now more relevant here:

https://docs.openshift.com/enterprise/3.1/install_config/upgrading/automated_upgrades.html#verifying-the-upgrade

That's for the "Automated Upgrade" path... for the "Manual Upgrade" path it doesn't look like we have an explicit "Verifying the Upgrade" section (there's a brief "verify" step at the end of "Upgrading Nodes"). Maybe we could introduce a "Verifying the Upgrade" section to the Manual topic as well.

**Comment by bfallonf** [comment]
@ahardin-rh Looks good to me!

**Comment by tpoitras** [comment]
LGTM! :+1: :fallen_leaf:

**Comment by sdodson** [comment]
Yes, the new content looks fine to me, may need to move it around as Alex suggests, i haven't kept up with the structure of that doc for a while so I'm not sure.

**Comment by ahardin-rh** [comment]
@adellape Edits complete. Good to go?

File: install_config/upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -278,3 +278,8 @@ common issues:
 [Note] Completed with no errors or warnings seen.
 ----
 ====
+
+Verify in your old *_/etc/sysconfig/openshift-master_* and
+*_/etc/sysconfig/openshift-node_* files that any custom configuration is added
+to your new *_/etc/sysconfig/atomic-openshift-master_* and
+*_/etc/sysconfig/atomic-openshift-node_* files.
```
Comment (adellape) [inline install_config/upgrading/automated_upgrades.adoc]: Sorry, I forgot to mention that the context of this is that it's only something you'll need to check when going from either:

OSE 3.0 --> 3.1

or

Origin 1.0 --> 1.1

Those were the upgrade paths that introduced these file name changes, but it was slightly different for OSE vs Origin. For OSE, they went from:

/etc/sysconfig/openshift-master --> /etc/sysconfig/atomic-openshift-master
/etc/sysconfig/openshift-node --> /etc/sysconfig/atomic-openshift-node

For Origin they went from:

/etc/sysconfig/openshift-master --> /etc/sysconfig/origin-master
/etc/sysconfig/openshift-node --> /etc/sysconfig/origin-node

So we'll need to 1) add something like "If you upgrade from (OSE 3.0 or Origin 1.0) to (OSE 3.1 or Origin 1.1), verify these files" and 2) make sure the file references are Origin/OSE appropriate.

Comment (adellape) [inline install_config/upgrading/automated_upgrades.adoc]: @sdodson Please correct me on the above re Origin if I'm off.

File: install_config/upgrading/manual_upgrades.adoc [hunk 2]
```diff
@@ -1166,3 +1166,54 @@ Modify the *_/etc/origin/master/scheduler.json_* file to add the `*kind*` and
...
+...
+[Note] Summary of diagnostics execution:
+[Note] Completed with no errors or warnings seen.
+----
+====
+
+Verify in your old *_/etc/sysconfig/openshift-master_* and
+*_/etc/sysconfig/openshift-node_* files that any custom configuration is added
+to your new *_/etc/sysconfig/atomic-openshift-master_* and
+*_/etc/sysconfig/atomic-openshift-node_* files.
```
Comment (adellape) [inline install_config/upgrading/manual_upgrades.adoc]: Regarding the placement of this in general, I think we should end the section w/ the suggestion about running the diagnostics tool (in both this Manual topic and in the equivalent Automated topic spot).

File: install_config/upgrading/automated_upgrades.adoc (hunk 1 above)
Comment (sdodson) [inline install_config/upgrading/automated_upgrades.adoc]: You're correct.

**Comment by ahardin-rh** [comment]
@adellape Fixed! PTAL. Thanks!

**Comment by adellape** [comment]
:+1:

**Comment by ahardin-rh** [comment]
**revhistory note:**

(for link:../install_config/upgrading.html[Upgrading])

    In the link:../install_config/upgrading/automated_upgrades.html#verifying-the-upgrade[Automated Upgrades] and link:../install_config/upgrading/manual_upgrades.html#manual-upgrades-verifying-the-upgrade[Manual Upgrades] sections, added guidance about verifying that custom configurations are added to the updated *_/etc_* paths after upgrading.

**Comment by adellape** [comment]
@ahardin-rh We can be more specific and say "the updated /etc/sysconfig/ paths". And possibly "after upgrading from OSE 3.0 to 3.1."

**Comment by ahardin-rh** [comment]
@adellape cool. Should I not mention the upgrade from Origin 1.0 to Origin 1.1 as well?

**Comment by adellape** [comment]
@ahardin-rh Correct, not needed because we don't do revhistory for Origin.

**Comment by ahardin-rh** [comment]
@adellape Gotcha. Fixed!  :boom:

**Pull request merged**